pkg/util/namespace/namespace.go

```diff
@@ -3,23 +3,69 @@ package namespace
 // Copyright (c) Microsoft Corporation.
 // Licensed under the Apache License 2.0.
 
-import (
-	"strings"
-)
-
-// IsOpenShift returns true if ns is an openshift managed namespace, including the default namespace
-func IsOpenShift(ns string) bool {
-	return ns == "" ||
-		ns == "default" ||
-		ns == "openshift" ||
-		strings.HasPrefix(ns, "kube-") ||
-		strings.HasPrefix(ns, "openshift-")
-}
-
-// IsOpenShiftSystemNamespace returns true if ns is an openshift managed namespace, without the default namespace
-func IsOpenShiftSystemNamespace(ns string) bool {
-	return ns == "" ||
-		ns == "openshift" ||
-		strings.HasPrefix(ns, "kube-") ||
-		strings.HasPrefix(ns, "openshift-")
+// FilteredOpenShiftNamespace returns true if ns is a namespace in the defined hardcoded map
+func FilteredOpenShiftNamespace(ns string) bool {
+	filteredNamespaces := map[string]struct{}{
+		"openshift":                                        {},
+		"openshift-apiserver":                              {},
+		"openshift-apiserver-operator":                     {},
+		"openshift-authentication-operator":                {},
+		"openshift-azure-logging":                          {},
```

Comment:
Maybe I am missing something, but wouldn't that be more explicit? Non-set keys will return false. So you can just do ```return filteredNamespaces[ns]```
```suggestion
	filteredNamespaces := map[string]bool{
		"openshift":                                              true,
	}
```